src/Router.js

```diff
@@ -1,11 +1,19 @@
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Footer from './components/Footer';
+import Top from './components/Top';
+import TopNav from './components/TopNav';
 import Detail from './pages/Detail/Detail';
+import SnkrsList from './pages/SNKRS/SnkrsList';
 
 function Router() {
   return (
     <BrowserRouter>
       <Routes>
         <Route path="/detail" element={<Detail />} />
+        {/* <Route path="/" element={<Top />} /> */}
+        <Route path="/" element={<TopNav />} />
+        {/* <Route path="/" element={<Footer />} /> */}
+        <Route path="/snkrs" element={<SnkrsList />} />
       </Routes>
     </BrowserRouter>
```

Comment:
```suggestion
    <BrowserRouter>
      <TopNav />
      <Routes>
        <Route path="/detail" element={<Detail />} />
        {/* <Route path="/" element={<Top />} /> */}
        <Route path="/"  />
        {/* <Route path="/" element={<Footer />} /> */}
        <Route path="/snkrs" element={<SnkrsList />} />
      </Routes>
    </BrowserRouter>
```

요거 TopNav는 모든 페이지가 공통적으로 사용하는 부분이니까 Routes 위에 나서 모든 페이지에 있게끔 하는게 어떤가요?

Reply:
이게 또 스니커즈 페이지 가면 없더라구요? 
어떻게할지 고민좀 해봐야 할 것 같습니다.